zipkin-ui/js/component_ui/trace.js

```diff
@@ -443,7 +448,11 @@ export default component(function trace() {
     this.on(document, 'uiZoomInSpans', this.zoomInSpans);
     this.on(document, 'uiZoomOutSpans', this.zoomOutSpans);
 
+    this.on(document, 'uiShowJson', this.showJson);
+
     const self = this;
+    self.raw = data.raw;
```

Comment:
good point.

Also, principle of least surprise wins for not defaulting to "raw" flag:
without disclaimers, it would confuse people when timestamps don't match
what the UI says.

I'm game for as-is.. I'd prefer that variable to be named something besides
raw (or clarified that it doesn't imply the raw parameter)